login_KA.feature

```diff
@@ -0,0 +1,144 @@
+Feature: log in Scoring 
+Background: 
+    Given installed and launched VPN on local PC
+    And a Gmail account
```

Comment:
I think there should be added some actions/additions, for example, 'user has a Gmail account' or something like this. Otherwise, it's not clear, what has to be done at this step